sheet4DiegoXavi.tex

```diff
@@ -0,0 +1,145 @@
+\documentclass[12pt,a4paper]{article}
+
+\usepackage{enumerate}
+\usepackage{graphicx}
+\usepackage{float}
+\usepackage[a4paper, margin=3.5cm]{geometry}
+\setlength{\parindent}{2em} 
+\setlength{\parskip}{1em} 
+\usepackage{amsfonts}
+\usepackage{amssymb}
+\usepackage{amsmath}
+\usepackage{comment}
+
+\newcommand{\cA}{\mathcal{A}}
+\newcommand{\cS}{\mathcal{S}}
+\newcommand{\RR}{\mathbb{R}}
+\newcommand{\CC}{\mathbbm{C}}
+\newcommand{\vol}{\text{vol}}
+\newcommand{\id}{\text{id}}
+
+\begin{document}
+
+\noindent
+\textbf{UPC MAMME / Discrete Geometry 2018}
+
+\noindent
+\textbf{Sheet 4}
+
+\noindent
+\textbf{Diego Acedo and Xavier Gombau}
+
+
+\subsubsection*{Solution 1}
+
+There is one thing which we do not find entirely clear in this problem. 
+The question whether we are given $P$ or not.
+
+If $P$ is not given, we have discussed with other colleagues the problem, and we have come to the conclusion that the lower faces of $Q^c$ are not determined from only the linear map $\pi : \RR^p \longrightarrow \RR^{q+1} $, and so it is not possible to give an algorithm to find such faces. 
+
+% Posibilidad de explicarlo mejor
+
+Nevertheless, we think that since $\pi$ is defined on $P$, then $P$ is given along with $\pi$, and so the lower faces are determined. 
+We have found a (computationally bad) algorithm which determines the set of lower faces of $Q^c$, but, since we cannot distinguish the interior points of $Q^c$ from those in the boundary, we use at some point a convex hull algorithm to solve this. 
+We are working on a way to solve that problem or show that indeed you are forced to use such convex hull algorithm. 
+
+Recall that the lower faces of $Q^c$ are those whose valid inequality is given by a hyperplane $h$ with $h_{d+1} < 0$ and a vector $h_0$, with $x \cdot h \leq h_0$ for all $x \in Q^c$.
+
+We do the following: 
+
+\begin{enumerate}
+	\item For each vertex $x_i \in P$, compute $\pi^c(x_i)$, so then it is $Q^c = conv\{\pi^c(x_i)\}$.
+	
+	\item Use a convex hull algorithm to find the actual vertices of $Q^c$.
+	
+	\item For all subsets of $d+1$ vertices:
```

Comment:
Why don't you iterate over all facets, which you found in the previous step?
I suggest you might need to specify what "running a convex hull algorithm" actually means for you.